src/assets/htmx.md

```diff
@@ -0,0 +1,347 @@
+---
+title: htmx
```

Comment:
try to find a title that is a bit more descriptive and draws the attention of the reader